components/Main/style.css

```diff
@@ -8,6 +8,88 @@
   overflow-x: auto;
   padding-bottom: 50px;
 }
+.main .project-name-container {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+}
+.three-dots-icon {
+  position: relative;
+  cursor: pointer;
+  margin: 0 0 3px 7px;
+}
+.menu-container {
+  position: relative;
+  cursor: pointer;
+}
+.actions-container {
+  position: absolute;
+  width: 102px;
+  height: 80px;
+  left: 16px;
+  top: 27px;
+  border-radius: 5px;
+  display: none;
+  filter: drop-shadow(0px 3px 3px rgba(0, 0, 0, 0.14));
+  background: #ffffff;
+}
+.show {
+  display: inline;
+}
+.hide {
+  display: none;
+}
+.actions-container .edit {
+  position: relative;
+  border-top-right-radius: 5px;
+  border-top-left-radius: 5px;
+  position: relative;
+
+  height: 40px;
+
+  cursor: pointer;
+  font-family: "Poppins";
+  font-size: 16px;
+  font-weight: normal;
+  font-style: normal;
+  text-align: center;
+  color: #6f6f6f;
+}
+.actions-container .edit:hover {
+  position: relative;
+
+  height: 40px;
+  background-color: rgba(0, 0, 0, 0.153);
+}
+.actions-container .edit p {
+  position: relative;
+  top: 12px;
+  left: -26px;
+}
+.actions-container .delete {
+  position: relative;
+  height: 40px;
+  border-bottom-right-radius: 5px;
+  border-bottom-left-radius: 5px;
+  font-family: "Poppins";
+  font-size: 16px;
+  font-weight: normal;
+  font-style: normal;
+  text-align: center;
+  color: #6f6f6f;
+  cursor: pointer;
+}
+.actions-container .delete p {
+  position: relative;
+  top: 13px;
+  left: -16px;
+}
+.actions-container .delete:hover {
+  position: relative;
+
+  height: 40px;
+  background-color: rgba(0, 0, 0, 0.153);
+}

```

Comment:
Reducing Redundancy:

Some styles, like `position: relative` and `height: 40px`, are repeated across `.edit`, `.delete`, and `.delete:hover`. Consider creating a shared class, such as `.action-item`, to hold these shared properties, and apply it to both `.edit` and `.delete`.

```css
/* Shared styles for action items */
.action-item {
    position: relative;
    height: 40px;
    cursor: pointer;
    font-family: "Poppins";
    font-size: 16px;
    font-weight: normal;
    text-align: center;
    color: #6f6f6f;
}

/* Additional styles for each specific element */
.actions-container .edit {
    border-top-right-radius: 5px;
    border-top-left-radius: 5px;
}

.actions-container .delete {
    border-bottom-right-radius: 5px;
    border-bottom-left-radius: 5px;
}

/* Hover effect */
.actions-container .edit:hover, .actions-container .delete:hover {
    background-color: rgba(0, 0, 0, 0.153);
}
```